### ISSUE TITLE
Non object in Ion_auth.php method 'logged_in()'

### ISSUE THREAD

**Issue by omarmd1986** [benedmunds/CodeIgniter-Ion-Auth#887]
![ion auth](https://cloud.githubusercontent.com/assets/14192756/11189671/433335d6-8c5f-11e5-90a1-1188bbf73b8a.png)

**Comment by benedmunds** [comment]
Try autoloading the session class.

**Comment by Adslive** [comment]
Hmm seems session helper problem, not the library.

**Comment by omarmd1986** [comment]
Yes the problem is the session library. I autload the session library, and the problm persist. Some other idea?

**Comment by benedmunds** [comment]
What version of CI are you on?

**Comment by omarmd1986** [comment]
version 3

**Comment by avenirer** [comment]
can you show us the session configuration you have? (application/config/config.php)

**Comment by omarmd1986** [comment]
[config.pdf](https://github.com/benedmunds/CodeIgniter-Ion-Auth/files/35872/config.pdf)

**Comment by avenirer** [comment]
Well... you didn't set the configuration correctly. You must set the $config['sess_save_path'] = NULL; to not be null but a path to a writable directory.

**Comment by omarmd1986** [comment]
i am not sure what do you say. mi sess_save_path is equal null, and my file system permission allowed full control in this directory. I don't know to do.

**Comment by avenirer** [comment]
@omarmd1986 I don't know if this helps you, but I would recommend you to follow one of my tutorials (if it's ok with @benedmunds ): http://avenir.ro/authentication-system-with-ion-auth-and-ci3/
